String freq

pr-th-m opened the pull request:
Just check the code guys

pr-th-m committed:
feat: created a String-Freq branch

pr-th-m committed:
feat: added the appropriate function

pr-th-m committed:
chore: added a jsDocumentation

KrithikaSuvarna01 commented on string.js:
```diff
@@ -1 +1,20 @@
+    {
+        console.log(`${letter} has be found ${count} times`);
+    }
+}
```
can you please call the function and check what value it returns!

KrithikaSuvarna01 reviewed (changes requested):
can you please call the function and check what value it returns!

pr-th-m committed:
feat: added a comment

pr-th-m committed:
feat: added a function

actually added a function here not in previous commit

feat: added the appropriate function

hrithik-b01 commented on hackerproject.js:
```diff
@@ -6,3 +6,22 @@
  * @param {string} str 
```
The function looks great, but the commit message doesn't follow our naming convention, can you please give appropriate commit message.

pr-th-m committed:
chore:added a oneline comment

NIREE23 committed:
Merge branch 'master' into String-Freq
